phpunit.xml.dist

```diff
@@ -9,7 +9,7 @@
          processIsolation="false"
          stopOnFailure="false"
          syntaxCheck="false"
-         bootstrap="tests/bootstrap.php">
+         bootstrap="vendor/autoload.php">
```

Comment:
I figured it's also a good time to remove `tests/bootstrap.php` and simplify the test bootstrapping.